### PR TITLE
[RF] POM 01.02.029 | <Header>Verify configure of case sensitive option in the Search box

### PR DESCRIPTION
**Pull request by DimaCurkanu**
https://trello.com/c/i7h4VO86/2013-rf-pom-0102029-headerverify-case-sensitive-option-in-the-search-box

### CODE REVIEW

**Comment by DimaCurkanu** [comment]
[[RF] POM 01.02.029 | <Header>Verify configure of case sensitive option in the Search box](https://trello.com/c/i7h4VO86/2013-rf-pom-0102029-headerverify-configure-of-case-sensitive-option-in-the-search-box)

File: cypress/pageObjects/HeaderAndFooter.js [hunk 1]
```diff
@@ -158,6 +159,16 @@ class HeaderAndFooter {
         this.getSearchBoxInputField().type('{enter}');
         return new BuiltInNodePage ();
         }
+
+    clickUserDropDownMenuConfigure() {
+        this.getUserDropDownMenuConfigure().click();
+            return new UserConfigurePage();
+        }
```
Comment (Rom-T) [inline cypress/pageObjects/HeaderAndFooter.js:166]: Delete please. This is a duplicate. We have a `selectUserConfigureMenu()`. If we select something from the menu, then the method must begin with the word `select`.

File: cypress/pageObjects/HeaderAndFooter.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ class HeaderAndFooter {
     getCurrentUserName = () => cy.get('.login .model-link span');
     getTitle = () => cy.get('head title');
     getListSearchResult = () => cy.get('.yui-ac-bd li').not('li[style="display: none;"]');
+    getUserDropDownMenuConfigure = () => cy.get('#yui-gen2');
```
Comment (Rom-T) [inline cypress/pageObjects/HeaderAndFooter.js:39]: Delete please. This is a duplicate. We have a `getUserConfigureMenu()`.

File: cypress/pageObjects/HeaderAndFooter.js [hunk 3]
```diff
@@ -158,6 +159,16 @@ class HeaderAndFooter {
...
+
+    clickUserDropDownMenuConfigure() {
+        this.getUserDropDownMenuConfigure().click();
+            return new UserConfigurePage();
+        }
+
+    typeSearchBoxInputFieldAndGoSearchResult(text) {
+        this.getSearchBoxInputField().clear().type(text + '{enter}');
+            return new ResultSearchBoxPage();
+        }
```
Comment (Rom-T) [inline cypress/pageObjects/HeaderAndFooter.js:171]: Delete please. This is a duplicate. We have a `searchTextSearchBox(text)`.

File: cypress/pageObjects/HeaderAndFooter.js (hunk 1 above)
Comment (DimaCurkanu) [reply cypress/pageObjects/HeaderAndFooter.js]: Done! Thank you!

File: cypress/pageObjects/HeaderAndFooter.js (hunk 2 above)
Comment (DimaCurkanu) [reply cypress/pageObjects/HeaderAndFooter.js]: Done! Thank you!

File: cypress/pageObjects/HeaderAndFooter.js (hunk 3 above)
Comment (DimaCurkanu) [reply cypress/pageObjects/HeaderAndFooter.js]: Done! Thank you!

**Review by Rom-T — APPROVED** [review]

File: cypress/e2e/tests/headerAndFooter.cy.js [hunk 4]
```diff
@@ -255,23 +255,37 @@ describe('headerAndFooter', () => {
...
+
+    it('AT_01.02.029 | Verify configure of case sensitive option in the Search box', () => {
+        headerAndFooter
+          .clickUserDropDownBtn()
+          .selectUserConfigureMenu() 
+          .clickSensitiveSearchCheckbox()
+          .clickUserConfigSaveBtn()
+        searchBoxDocumentationPageData.dataCapCase.forEach(($el,idx) => {      
+           headerAndFooter
+                .searchTextSearchBox(searchBoxDocumentationPageData.dataCapCase[idx])
```
Comment (IrynaKolh) [inline cypress/e2e/tests/headerAndFooter.cy.js:286]: нужно добавить .clear() в метод

 searchTextSearchBox(text) {
        this.getSearchBoxInputField().clear().type(text + '{enter}');
        return new ResultSearchBoxPage();
    }

Comment (DimaCurkanu) [reply cypress/e2e/tests/headerAndFooter.cy.js:286]: Done! Thank you!

**Review by IrynaKolh — APPROVED** [review]

**Pull request merged**